.github/CODEOWNERS

```diff
@@ -0,0 +1,50 @@
+###################################################################################################
+# Overview
+#
+# Pattern used to match files follows most of the same rules as used in gitignore files. Order is
+# important; the last matching pattern takes precendence.
+#
+# For more info see:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+###################################################################################################
+
+
+###################################################################################################
+# Global (@keycloak/maintainers)
+###################################################################################################
+
+/**                                                         @keycloak/maintainers
+
+###################################################################################################
+# Testing (@keycloak/testing-maintainers)
+###################################################################################################
+
+/testsuite/**                                               @keycloak/testing-maintainers
```

Comment:
@stianst changes LGTM, but I have one small comment on it. Following our plan of ownership over test failures, I believe that it makes sense for the `/testsuite/**` to be owned by `keycloak/maintainers`.

The motivation for that is to give our teams ownership over it for improvements and bug fixes.

Reply:
Not following, the reason for testing-maintainers is to workaround the fact that most PRs need an update in testsuite today, so we're allowing any team to review/merge changes to the testsuite. If we change this to keycloak/maintainers then there's almost no change with this PR as a global maintainer will need to sign-off on any PR affecting the testsuite, which is a high proportion.

Reply:
My apologies @stianst, that's my fault. What you described is exactly the behavior I expected. Thank you.